MANIFEST.in

```diff
@@ -1 +1,2 @@
+recursive-include lightbeam/tests *.jsonl *.json *.yaml *.tsv
```

Comment:
This should ensure that the data and configuration needed for the test suite are included in the lightbeam package.